src/app/cli/cli_processor.cpp

```diff
@@ -271,6 +271,12 @@ void CliProcessor::process(Context* ctx)
           if (m_exporter)
             m_exporter->setTrimCels(true);
         }
+        // --trim-by-grid
+        else if (opt == &m_options.trimByGrid()) {
+          cof.trimByGrid = true;
+          if (m_exporter)
+            m_exporter->setTrimByGrid(true);
```

Comment:
As we talked in the email, I think here we can call `m_exporter->setTrimCels(true);` too.